src/clj/schema/macros.clj

```diff
@@ -201,13 +220,17 @@
      (or (:always-validate fn-meta)
          *assert*))))
 
+(def ^:dynamic *ufv-sym* 'ufv__) ;; dynamic for backwards compat
```

Comment:
I think it's worth a docstring since it's not obvious to me what's going on here.  Also maybe worth elaborating on the backwards compat concern.

Reply:
Thanks for bringing this up. It's a little more hygienic to use fresh symbols to let-bind variables. eg., `(s/fn [ufv__])` would capture the binding.


In terms of implementation, I didn't want to add an extra arity to `process-fn-arity` because 6 args seemed too much, but I'm considering adding a 1-arg arity that just takes a map. That will obviate the need for this dynamic var.